src/components/templates/Jazzbar/JazzTab/JazzTab.scss

```diff
@@ -49,8 +77,7 @@
         background-color: #19181a;
         overflow: hidden;
         position: relative;
-        // @debt Thomas' magic number. Should result in the best 16-9 ratio for Monday presentation
-        height: 333px;
+        height: 100%;
```

Comment:
Do we still account for the 16:9 needs here?

Reply:
The video sizes has changed and with new hardcoded video sizes(not the best way, but the most optimal one), we don't need to hardcode the video size

Reply:
When you say 'with new hardcoded video sizes', where is this done? Within the media queries, or?

Reply:
In local constants. Basically, we have video height as 2 * video participant height (195px) + paddings. It's hard to keep the best 16:9 aspect ratio for every screen size, but it's good enough for all usecases now. This view was approved by @Sparkle-T, @sparkleed

Reply:
Cool cool, makes sense.